Changes made to the AIHD Number

ItsMurumba opened the pull request.

ItsMurumba committed:
1. Added a method to set Telephone Number to preferred and the rest to not preferred.
2. Set automatic close of visits everyday at midnight.

ItsMurumba committed:
Fixed the issue of the preffered Identifier. The sql query was throwing an error and thus introduced a condition. The Update works only if we have wrong set identifiers.

ItsMurumba committed:
1. Commented out the preferred function on start.
2. Made changes on validation error color
3. Changes to the AIHDNumber generation

ItsMurumba committed:
Merge branch 'master' of https://github.com/IntelliSOFT-Consulting/AIHD

ItsMurumba committed:
Merge branch 'master' of https://github.com/IntelliSOFT-Consulting/AIHD

ningosi commented on api/src/main/java/org/openmrs/module/aihdconfigs/AihdConfigurationsActivator.java:
```diff
@@ -50,211 +50,223 @@
+        AdministrationService as = Context.getAdministrationService();
+
+        as.executeSQL("UPDATE patient_identifier SET preferred=0 WHERE identifier_type in (1,2,3,4) AND preferred=1;", true);
+        as.executeSQL("UPDATE patient_identifier SET preferred=1 WHERE identifier_type=5 AND preferred=0;;", true);
```
We have 2 semi colons

ItsMurumba replied:
I see this let me correct

ItsMurumba committed:
Commented out the method to set preferred identifier on module startup

ItsMurumba committed:
1. Add foot clinic fields as a section under the Physical exam section
2. No concept IDs supplied

ItsMurumba committed:
Added the relevant field to the initial form but with no concept IDs

ItsMurumba committed:
1. Added admission section in page 2 (Were you admitted since the last visit? date of admission, date of discharge, admission details)
2. Added foot clinic fields as a section under the exam section
3. Renamed "Referrals" section to "follow up Plan"
4. Added “Admit patient” checkbox & reason for admission in the follow up section)

ItsMurumba committed:
Added the concept ID for the additional fields as per the realease 2.0 plans.

ItsMurumba committed:
Changed the FAQs from docs to PDF

ItsMurumba committed:
Wrote a method to update the Facilities with repective subcounty and counties. Created a separate facilitiesForUpdate.csv for the method to work.

ItsMurumba committed:
Added facility levels to the facilitiesForUpdate.csv and saving of the level to the address13 of the location properties.

ItsMurumba committed:
Started working on the carry over of drugs from initial and follow up forms to the sebsequent visits. Incomplete
